src/views/Qafi/index.jsx

```
@@ -16,7 +16,7 @@ import {
   searchQafi,
 } from "../../app/features/qafi";
 import { AddQafi, QafiPreviewer } from "../../services/qafi";
-const Qafi = () => {
+const Qafi = ({ isDashboard = false }) => {
```

ESLint

[error] 'isDashboard' is missing in props validation

(react/prop-types)